src/components/SampleItem.tsx

```diff
@@ -0,0 +1,393 @@
+import React from 'react'
+import { Button, Col, Popover, Row, Slider, Space, Switch } from 'antd'
+import { CloseSquareFilled, SettingOutlined } from '@ant-design/icons';
+import { FaEye, FaEyeSlash } from 'react-icons/fa'
+import Description from './Description'
+import * as dmv from 'dicom-microscopy-viewer'
+import * as dcmjs from 'dcmjs'
+
+import SamplesList from './SamplesList'
+
+interface SampleItemProps {
+  opticalPathSequence: dmv.metadata.OpticalPathDescription,
```

Comment:
The use of the term `opticalPathSequence` for the property is very confusing, since it is actually only an individual item.

```suggestion
  opticalPathDescription: dmv.metadata.OpticalPathDescription,
```